lang/en.js

```diff
@@ -10,8 +10,9 @@ export default {
 	"components.form-element.defaultError": "{label} is invalid.",
 	"components.form-element.defaultFieldLabel": "Field",
 	"components.form-element.input.email.typeMismatch": "Email is not valid",
-	"components.form-element.input.number.rangeOverflow": "Number must be lower than {max}",
-	"components.form-element.input.number.rangeUnderflow": "Number must be higher than {min}",
+	"components.form-element.input.number.rangeError": "Number must be between {min} and {max}.",
```

Comment:
I'm not sure if its a strict requirement but should these be added with an empty string in the other languages?

Reply:
Would it hurt to add them? If not, then I can add them in this PR.

Reply:
It definitely won't hurt, [it seems to be the approach we take most often](https://github.com/BrightspaceUI/core/commit/2fbdc454b0a6fd022947cbedfb5bdd5999ea2eb1#diff-d91f9f7171040f8d75fd69c48184da63). I'm not sure if its a strict requirement but probably worth doing just incase.